### ISSUE TITLE
ts_eval definition

### ISSUE THREAD

**Issue by jonpage** [bentut/udaman#7]
The definition for `ts_eval` does not appear in the repository. I can't run REBUILD.rb and other important stuff without it.

**Comment by bentut** [comment]
That's weird. It should be in one of the initializer files. Probably
stringextension.rb. And I think it only works if iu do .ts_eval=

On Tuesday, March 18, 2014, Jonathan Page notifications@github.com wrote:

> The definition for ts_eval does not appear in the repository. I can't run
> REBUILD.rb and other important stuff without it.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/bentut/udaman/issues/7
> .

## 

Ben Trevino....
808-225-4725

UHERO
The Economic Research Organization at the University of Hawaii
www.uhero.hawaii.edu | www.facebook.com/UHERO.forum
Database Manager
btrevino@hawaii.edu

INTERISLAND TERMINAL
Presenting international contemporary art, design and film /
Advancing the role of the arts in innovation
www.interislandterminal.org

**Comment by jonpage** [comment]
Yeah, it looks like REBUILD.rb is using the correct syntax. I don't see stringextension.rb in the repo. Is it in another gem?

**Comment by bentut** [comment]
It should be this file:
https://github.com/bentut/udaman/blob/master/config/initializers/string_extension.rb

On Tuesday, March 18, 2014, Jonathan Page notifications@github.com wrote:

> Yeah, it looks like REBUILD.rb is using the correct syntax. I don't see
> stringextension.rb in the repo. Is it in another gem?
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/bentut/udaman/issues/7#issuecomment-37991693
> .

## 

Ben Trevino....
808-225-4725

UHERO
The Economic Research Organization at the University of Hawaii
www.uhero.hawaii.edu | www.facebook.com/UHERO.forum
Database Manager
btrevino@hawaii.edu

INTERISLAND TERMINAL
Presenting international contemporary art, design and film /
Advancing the role of the arts in innovation
www.interislandterminal.org
